harvest/wizard/_base.py

```diff
@@ -0,0 +1,101 @@
+# Builtins
```

Comment:
I would say this module is well documented, but if there is anything that looks off let me know.